pyproject.toml

```diff
@@ -369,7 +369,7 @@ furo = ">=2022.12.7,<2024.0.0"
 # system changes.
 # We are happy to raise these upper bounds upon request,
 # provided we check that it's safe to do so (i.e. that CI passes).
-requires = ["poetry-core>=1.1.0,<=1.6.0", "setuptools_rust>=1.3,<=1.6.0"]
+requires = ["poetry-core>=1.1.0,<=1.6.0", "setuptools_rust>=1.3,<=1.6.0", "wheel"]
```

Comment:
Context? Related to [internal conversation](https://matrix.to/#/!SGNQGPGUwtcPBUotTL:matrix.org/$0DWniW6yzTa7SCfyiQDiSAnwjEomKE6Rs3HJiU9Z9O0?via=jki.re&via=matrix.org&via=element.io)?

Reply:
(This looks really unusual---are there examples of other python projects explicitly depending on `wheel` as a build requirement?)

Reply:
(If this is worth landing it ought to be a separate PR)

Reply:
This is a mistake of mine sorry. I experimented when having a trouble with Python 3.11 install, and some `git commit -a` went through a bit too quickly.